lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -268,6 +268,20 @@ LazyNativeFunctions::convolution_backward_overrideable(
     const at::Tensor& weight, at::IntArrayRef stride, at::IntArrayRef padding,
     at::IntArrayRef dilation, bool transposed, at::IntArrayRef output_padding,
     int64_t groups, std::array<bool, 3> output_mask) {
+  // Lower to cudnn_convolution_backward when possbile
+  if (at::globalContext().userEnabledCuDNN() &&
```

Comment:
Do you think we can also have a similar reduced story for CPU, like supporting the most common one?

Reply:
It'd be good to confirm that the cudnn conv we picked is indeed useful for the nvfuser team since a main motivation for doing the short term fix was to unblock their channels-last work, and also get a read on how soon Joel thinks refactored convolution will be ready.

We probably have bigger fish to fry than supporting more convs manually, but we can discuss the tradeoffs if there is some use case someone wants to enable.

Reply:
My intent is that if we can have at least one CPU conv that can be used by TorchBench's resnet50, then we can have some early performance numbers.

Reply:
CPU has more kernels to be called, conv2d/conv3d, transposed/non-transposed, group/no-group, etc. I don't think it's worth to add it in this temp fix. I think we can focus on GPU for now. Even with our nvfuser friends looking at perf, there are plenty of space and benchmarks left for us to investigate on GPU.